.github/workflows/linux.yml

```diff
@@ -346,3 +346,105 @@ jobs:
           path: |
             /tmp/tinygo.linux-arm.tar.gz
             /tmp/tinygo_armhf.deb
+  build-linux-arm64:
+    # Build ARM64 Linux binaries, ready for release.
+    # It is set to "needs: build-linux" because it modifies the release created
+    # in that process to avoid doing lots of duplicate work and to avoid
+    # complications around precompiled libraries such as compiler-rt shipped as
+    # part of the release tarball.
+    runs-on: ubuntu-18.04
+    needs: build-linux
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
This can be removed. It slows down the build and the submodules that are actually needed (binaryen) are pulled specifically when needed.